liminal/core/util/class_util.py

```diff
@@ -61,10 +60,11 @@ def import_module(package, recursive=True):
         package = importlib.import_module(package)
     results = {}
     for loader, name, is_pkg in pkgutil.walk_packages(package.__path__):
-        full_name = package.__name__ + '.' + name
-        results[full_name] = importlib.import_module(full_name)
-        if recursive and is_pkg:
-            results.update(import_module(full_name))
+        if not name == 'liminal_python_server':
```

Comment:
feels like a workaround. why this if?

Reply:
@aviemzur

Reply:
Yes, let's remove this line @zionrubin 
It was needed in previous version of liminal python server but we since removed it